run-build.sh

```diff
@@ -93,11 +93,18 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$2}" )
             shift
             ;;
+        --env-vars)
```

Comment:
@jyoungyun Could you describe the use case for this? My first impression is that this could be solved using an  external script that sets the needed variables rather than having the CLI's scripts take on this responsibility, but I'm probably missing some of the context.

Thanks.